### PR TITLE
Make demo show UTF-8 chars for a single result & fixing to work with mvn

### PR DESCRIPTION
**Pull request by lararennie**
This change is [Reviewable](https://reviewable.io/reviews/googlei18n/libphonenumber/1646)

### CODE REVIEW

**Comment by lararennie** [comment]
Fixes among other things issue #1419

**Comment by lararennie** [comment]
Note: it is quite hard to run the demo at the moment, the README instructions using maven don't work, and using ant you have to manually copy the libs & remember to set the java version to 1.7 before you compile & copy the app-engine sdk into the right place.

Wondering if we want to update the README to at least explain the ant way.

**Comment by lararennie** [comment]
Mm the maven problem (present with no change) seems to prevent the continuous build from working too :/

File: java/demo/pom.xml [hunk 1]
```diff
@@ -1,67 +1,21 @@
 <?xml version="1.0"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
+  <groupId>com.googlecode.libphonenumber</groupId>
+  <artifactId>demo</artifactId>
+  <version>8.3.4-SNAPSHOT</version>
+  <packaging>jar</packaging>
```
Comment (andyst) [inline java/demo/pom.xml]: So adding <packaging> and dropping the <repositories> did the trick?
Thanks for figuring that out!

**Comment by lararennie** [comment]
The bit that fixed it seemed to be removing the "repositories" section. It seems to be bad practice to specify it unless you have your own private one (we don't).

Yes will update description!

**Comment by SumathiSundara** [comment]
Fixed in release 8.4.0

**Pull request merged**